doc/api/buffer.md

```diff
@@ -38,11 +38,11 @@ const buf3 = Buffer.allocUnsafe(10);
 // Creates a Buffer containing [0x1, 0x2, 0x3].
 const buf4 = Buffer.from([1, 2, 3]);
 
-// Creates a Buffer containing ASCII bytes [0x74, 0x65, 0x73, 0x74].
-const buf5 = Buffer.from('test');
```

Comment:
Why remove this example? I think I can see where this is coming from, but it might be better to just replace `ASCII` with `ASCII/UTF-8` in the comment?